src/styles/index.css

```diff
@@ -0,0 +1,111 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+h2, p {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  padding: 32px 16px 17px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  align-self: center;
+}
+
+.card__title {
+  padding: 40px 0 4px;
+  color: #060B35;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px; /* 150% */
+}
+
+.card__subtitle {
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px; /* 140% */
+}
+
+.card__btn {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background: #00ACDC;
+  color: #FFF;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px; /* 114.286% */
+  text-transform: uppercase;
+  cursor: pointer;
+  border: none;
+}
+
+.card__btn:hover {
+  background: #FFF;
+  color: #00ACDC;
+  border: 1px solid #00ACDC;
+}
+
+.grade {
+  display: flex;
+  justify-content: space-between;
+  padding: 16px 0 24px;
+}
+
+.grade__stars {
+  display: flex;
+}
+
+.grade__star {
+  background: url("../images/star-active.svg") 50% no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.grade__star:last-child {
```

Comment:
Consider implementing a logic from the previous task. Active stars should depend on the modifier class